event.json

```diff
@@ -0,0 +1,14 @@
+{
+  "pull_request": {
+    "number": 1,
+    "title": "Test PR",
```

Comment:
The title 'Test PR' is generic. For better clarity and tracking, consider using a more descriptive title that reflects the purpose of the pull request.